src/openapi/streaming/subscription.js

```diff
@@ -441,6 +457,12 @@ function Subscription(streamingContextId, transport, serviceGroup, url, subscrip
     }
     this.connectionAvailable = true;
 
+    // Defines current number of fallback requests done due to invalid format.
+    this.formatFallbackCount = 0;
+
+    // Defines maximums fallback requests possible for invalid format.
+    this.maxFormatFallbacks = 1;

```

Comment:
At the moment don't see a reason for falling back more then once as part of specific subscription instance. 
Maybe instead of counters, simple flag in such case would be enough?

Need to also check if modify can bring protobuf format once again. AFAIK, it shouldn't (should only override part of arguments). However it might by that GO store will still hold protobuf format and pass it over on modify. Will validate such case.

Reply:
I'm not sure we need anything?
We already explicitly only fallback if the format is protobuf and we fallback to json, so isn't that enough?

Reply:
Should be enough. Just wanted to guard against openapi issue where they would wrongly return UnsupportedSubscriptionFormat error. We had issues in auth, which spammed us with retries due to invalid constant auth errors from openapi. 

Howevere if you believe that in this case it's to much, i can drop it. It's for sure less probable issue to happen then what we encountered with auth retry loop.